app/Mails/Template/Template.php

```diff
@@ -7,9 +7,9 @@
 final class Template
 {
     public function __construct(
-        public string $type,
+        public TemplateName $name,
         public int $id,
-        public bool $enabled,
+        public bool $enabled = true,
```

Comment:
Was there a good reason to give this a default value?

Reply:
I prefer a reason related to some added business values instead of easier tests. So if you don't mind that would to great yes.